File: module1/src/main/java/com/moodstrikerdd/module1/debug/MainActivity.java
package com.moodstrikerdd.module1.debug;

import com.moodstrikerdd.lib_common.base.BaseActivity;
import com.moodstrikerdd.module1.Module1Fragment;
import com.moodstrikerdd.module1.R;

/**
 * @author moodstrikerdd
 */
public class MainActivity extends BaseActivity {

    @Override
    public int getLayoutId() {
        return R.layout.module1_activity_main;
    }

    @Override
    public void initData() {

    }

    @Override
    public void initView() {
        getSupportFragmentManager().beginTransaction()
                .add(R.id.flContain, new Module1Fragment(),Module1Fragment.class.getSimpleName())
                .commit();
    }
}

File: module3/src/main/java/com/moodstrikerdd/module3/debug/MainActivity.kt
package com.moodstrikerdd.module3.debug

import com.moodstrikerdd.lib_common.base.BaseActivity
import com.moodstrikerdd.module3.Module3Fragment
import com.moodstrikerdd.module3.R

class MainActivity : BaseActivity() {
    override fun getLayoutId() = R.layout.module3_activity_main

    override fun initData() {
    }

    override fun initView() {
        supportFragmentManager.beginTransaction()
                .add(R.id.flContain, Module3Fragment(), Module3Fragment::javaClass.name)
                .commit()
    }

}
